dev.neovide.neovide.yml

```diff
@@ -0,0 +1,53 @@
+id: dev.neovide.neovide
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+# sdk-extensions:
+#   - org.freedesktop.Sdk.Extension.rust-stable
+command: neovide
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri # opengl
+  - --share=network
+  - --share=ipc
+  - --filesystem=host
+  - --filesystem=xdg-config/neovide
```

Comment:
Needs exception https://docs.flathub.org/docs/for-app-authors/linter#finish-args-unnecessary-xdg-dir-access

Reply:
So does config really need to be shared with host? You can just copy-paste to migrate

Reply:
Yes users could definitely copy-paste (or sym-link?) their config to the flatpak volume. That being said, I think most users would expect to be able to follow the vendor [instructions](https://neovide.dev/config-file.html) and put their config in `$XDG_CONFIG_HOME/neovide/config.toml`